ui/packages/consul-ui/app/components/topology-metrics/card/index.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+import { get } from '@ember/object';
+
+export default class TopologyMetrics extends Component {
+  // =methods
+  get hrefPath() {
+    const source = get(this.args.item, 'Source') || '';
```

Comment:
I have the same question here about falsey-ness as the last PR, will Source ever be falsey?

Reply:
let me verify this with BE

Reply:
Source always return a value. It can't be falsy. Removing `|| ''`;